firmware/library/L3_HAL/sddriver.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+#include <cstdint>
+
+// Relevant Datasheet: https://www.sdcard.org/downloads/pls/pdf/index.php?p=Part1_Physical_Layer_Simplified_Specification_Ver6.00.jpg&f=Part1_Physical_Layer_Simplified_Specification_Ver6.00.pdf&e=EN_SS1
+class SdInterface {
+ public:
+    // TODO(#152): define a command frame struct
+    struct {
```

Comment:
Define the struct and then define its variable. Also, does this need to be packed? Are you transmitting this struct down the wire?